src/U1Manager.cpp

```
@@ -0,0 +1,397 @@
+    uint32_t now = millis();
+    if (moonrakerBackoffUntilMs_ != 0 && (int32_t)(now - moonrakerBackoffUntilMs_) < 0) {
+        return;
+    }
+
+    if (sync.is_uid_lookup) {
+        // Generic UID tag (NFC+) — Spoolman is the only data source. Single POST.
+        U1FilamentInfo info = buildFromSpoolmanSync(sync, state);
+        int code = postFilamentDetectSet(channel, info);
+        Serial.printf("U1Manager: publishFromSpoolmanSync(UID) channel=%u spool=%d — HTTP %d\n",
+                      (unsigned)channel, sync.spoolman_id, code);
+        if (code < 0 && code != -1000 && code != -1001) {
```

Clang (14.0.6)

[warning] repeated branch in conditional chain

(bugprone-branch-clone)